internal/dataplane/parser/translators/httproute.go

```diff
@@ -369,10 +403,21 @@ func GeneratePluginsFromHTTPRouteFilters(filters []gatewayapi.HTTPRouteFilter, p
 		case gatewayapi.HTTPRouteFilterResponseHeaderModifier:
 			kongPlugins = append(kongPlugins, generateResponseHeaderModifierKongPlugin(filter.ResponseHeaderModifier))
 
-		case gatewayapi.HTTPRouteFilterExtensionRef,
-			gatewayapi.HTTPRouteFilterRequestMirror,
+		case gatewayapi.HTTPRouteFilterExtensionRef:
+			plugin, err := generateExtensionRefKongPlugin(filter.ExtensionRef)
+			if err != nil {
+				return nil, "", err
+			}
+			if len(pluginsAnnotation.String()) > 0 {
+				fmt.Fprintf(&pluginsAnnotation, ",%s", plugin)
```

Comment:
```suggestion
				_, err := pluginsAnnotation.WriteString(","+plugin)
				if err != nil {
					return nil, "", err
				}
```

https://pkg.go.dev/strings#Builder.WriteString